### ISSUE TITLE
register h2, http/1.1 handler for optional router web bindings when same port as ziti-edge

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti#1808]
Presently, it's necessary to configure router health checks on a discrete web binding because a web binding that incidentally uses the same port as the ziti-edge or link listener will fail "silently" (a non-fatal event is logged). 

While that's an acceptable configuration for someone that knows precisely what they're doing, registering a standard handler for aggregated web bindings would enable a more useful default run state.

I'm converging on a default run state where the ziti CLI generates router configs from the built-in templates, including a web binding for health-checks on the same port as the singular port already having registered handlers for ALPN ziti-edge, ziti-link.

```
Mar 09 19:57:37 mira entrypoint.bash[879073]: [  47.246]   ERROR transport/v2/tls.(*sharedListener).processConn [tls:0.0.0.0:3022]: {remote=[127.0.0.1:48176] error=[not handler for requested protocols [h2 http/1.1]]} handshake failed
```

**Comment by qrkourier** [comment]
TL;DR healthchecks are unavailable when web binding is same port as edge listener

**Comment by qrkourier** [comment]
We discussed that single-port router health checks are unnecessary because binding health checks on a separate port is more secure.

The other impetus for this change request was fronting a router's WSS edge listener with a TLS proxy, i.e., one that provides a  trusted server certificate for ZBR, but we decided it's simpler to explain, document, etc. to say "always use `identity.alt_server_certs`, and don't front any Ziti TLS listeners with a TLS-terminating proxy."
